include/hal/library/platform_lib.h

```diff
@@ -23,6 +23,11 @@ void libspdm_sleep(uint64_t milliseconds);
  *
  * @param microseconds     The time interval for which execution is to be suspended, in milliseconds.
  *
+ * Be aware if 0 is given as the sleep time interval, depends on the platform implementation though,
```

Comment:
I will change it as below:
1. remove such description in `include/hal/library/platform_lib.h`
2. add if `microseconds` is 0 then return immediately in windows implementation and explain why.
3. remove `microseconds > 0` check from all calling positions.

We're good on linux implementation since man page of `select()` says
> If both fields of the timeval structure are zero, then **select()** returns immediately.